app/assets/stylesheets/theme/custom.css.scss

```diff
@@ -1,17 +1,15 @@
-select {
-  &#search_category {
-    background: url(asset-path("theme/dropdown_arrow.png")) no-repeat 95% center transparent;
-    background-size: 10px 5px;
-    border: none;
-    box-shadow: none;
-    font-weight: 400;
-    height: auto;
-    padding: 10px 22px 10px 4px;
-    width: 160px;
-    -webkit-appearance: none;
-    -moz-appearance: none;
-    appearance: none;
-  }
+select.form-control.search_options {
```

Comment:
Rule set contains (11/10) properties<br>Avoid qualifying class selectors with an element.<br>Selector `search_options` should be written in lowercase with hyphens